Graph-a-Single-SNMP-OID.md

```diff
@@ -34,5 +34,9 @@ When finished filling in values for these fields, click the Create button. Your
 new graph can now be accessed through the Graph Management page or the Graphs
 tab inside of Cacti.
 
+#### Note on graphing single OID's with spine
+Spine will only proccess a numeric OID i.e 1.3.6.1.1.1 spine will not parse OID's such as enterprise.1.3.6.1.1.1 
+Otherwise the spine poller will not proccess the OID
```

Comment:
These two lines need some better wording as it seems repetitive and one continuous line.

Reply:
fixed